app/uk/gov/hmrc/exports/services/CountriesService.scala

```diff
@@ -41,7 +41,8 @@ class CountriesService @Inject()() {
 
   private def countryCode: String => String = cc => cc.split(":")(1).trim
 
-  def findCountryCode(countryName: String): String = allCountries.find(_.countryName == countryName).map(_.countryCode).getOrElse("")
+  def findCountryCodeOrReturnCountryName(countryName: String): String =
+    allCountries.find(_.countryName == countryName).map(_.countryCode).getOrElse(countryName)
```

Comment:
Wouldn't be better to return `Option[String]` and let the client decide what to do in case of not finding the country code?

Reply:
It is used in Java class. The normal `getOrElse` doesn't work there

Reply:
I tried to prepare specific method in Java, but with my knowledge it's difficult to use scala list with countries (case class Country) to do this inside the Java class

Reply:
If you have any ideas please let me know :) I don't like this solution, but we need to make something to protect the data

Reply:
Could go old school and return null!  Or how about the client provides a default value if no mapping found?